waspc/data/Generator/templates/server/src/auth/providers/email/signup.ts

```diff
@@ -24,12 +24,12 @@ export function getSignupRoute({
     fromField,
     clientRoute,
     getVerificationEmailContent,
-    allowUnverifiedLogin,
+    isEmailAutoVerified,
 }: {
     fromField: EmailFromField;
     clientRoute: string;
     getVerificationEmailContent: GetVerificationEmailContentFn;
-    allowUnverifiedLogin: boolean;
+    isEmailAutoVerified: boolean;
```

Comment:
Why is this a variable coming from outside? Why not directly have that logic here inside this function? Because you wanted to templatizing this file? In that case ok, otherwise it would make more sense to me to put it inside here.